.github/workflows/main.yaml

```diff
@@ -5,8 +5,6 @@ on:
     branches:
       - main
   push:
-    branches:
-      - main
```

Comment:
what is this for?

Reply:
This ensures CI will run on push to a branch. Before this change, CI would only run after PR creation.

Reply:
I like that change.

Reply:
It's useful if someone likes to push commits while working on the changes, but doesn't necessarily want to create a PR for something that's not ready.